### ISSUE TITLE
Dataframe - Can't drop and rename

### ISSUE THREAD

**Issue by Mbensler** [opensource9ja/danfojs#239]
**Describe the bug**

I am trying to read a .xlsx file, then drop three columns, and then rename the remaining columns.
If i remove the code for dropping, renaming works just fine, and if i remove the code for renaming, dropping works just fine.
HOWEVER they don't work together!
```js
c:\Users\mbe\Documents\dev\sjr_webapp\node_modules\danfojs-node\dist\core\frame.js:1769
      delete self[name];
`(node:8272) UnhandledPromiseRejectionWarning: TypeError: Cannot delete property 'Tilstand' of [object Object]`
```

Renaming first and dropping second, doesn't work either.

```csv
Tilstand,Kode,Koncernkode,Driftsform,CVR,Navn,,Afdeling,Kunde ansv.,Leds. ansv.,Regnskabsafslutning
Aktiv,0005,,Personlig virksomhed,22278613,Søren's Udlejning I/S,,Padborg,Bent Martin Haugaard,Claus Thomsen,31 december
```

```js
  let df = await dfd.read_excel(clientlist, { header: false });
  df.drop({
    columns: ['Koncernkode', '', 'Regnskabsafslutning'],
    axis: 1,
    inplace: true,
  });
  df.rename({
    mapper: {
      Tilstand: 'is_active',
      Kode: 'no',
      Driftsform: 'type',
      CVR: 'cvr_cpr',
      Navn: 'name',
      Afdeling: 'department',
      'Kunde ansv.': 'customer_responsible',
      'Leds. ansv.': 'management_responsible',
    },
    inplace: true,
  });
  df.print();
```

**To Reproduce**
Copy paste my code

**Expected behavior**
I expect to be able to drop columns, and then rename the remaning columns after.

**Comment by steveoni** [comment]
@Mbensler sorry for the late reply.
I was able to reproduce the error. but from my end, the changes were reflected in the table. I will check what's causing the error.

**Comment by steveoni** [comment]
@Mbensler this now fixed in #247
